src/index.js

```diff
@@ -101,7 +101,7 @@ export default function(babel) {
     const arg = path.node.arguments[0];
     if (t.isNumericLiteral(arg) ||
         (t.isStringLiteral(arg) && reInteger.test(arg.value))) {
-      return t.bigIntLiteral(`${ arg.value }`);
+      return t.bigIntLiteral(arg.value);
```

Comment:
Just so you know the tests are failing with this change because `bigIntLiteral` takes a string value and sometimes `arg.value` is a number. This is why I kept the template literal initially.

Reply:
Gotcha, I had missed that nuance. Note that CI was already failing before this change

Reply:
```suggestion
      return t.bigIntLiteral(`${ arg.value }`);
```